### ISSUE TITLE
[FEATURE REQUEST] ParaCLAP

### ISSUE THREAD

**Issue by bagustris** [felixbur/nkululeko#208]
**Why do you propose this feature?**
From the abstract of the paper
> Contrastive language-audio pretraining (CLAP) has recently emerged as a method for making audio analysis more generalisable. Specifically, CLAP-style models are able to `answer' a diverse set of language queries, extending the capabilities of audio models beyond a closed set of labels. However, CLAP relies on a large set of (audio, query) pairs for pretraining. While such sets are available for general audio tasks, like captioning or sound event detection, there are no datasets with matched audio and text queries for computational paralinguistic (CP) tasks. As a result, the community relies on generic CLAP models trained for general audio with limited success. In the present study, we explore training considerations for ParaCLAP, a CLAP-style model suited to CP, including a novel process for creating audio-language queries. We demonstrate its effectiveness on a set of computational paralinguistic tasks, where it is shown to surpass the performance of open-source state-of-the-art models.

Since Nkululeko is mostly for paralinguistic prediction from speech, why don't we add this feature (will be interesting to compare with emotion2vec and others)?

Hugging face page: https://huggingface.co/KeiKinn/paraclap

**Describe the solution you'd like**
```ini
[FEAT]
type = ['paraclap']
```

**Comment by felixbur** [comment]
ok, clap is already there, so i guess that's easy
